couchdb/10-docker-default.ini

```diff
@@ -18,6 +18,15 @@ bind_address = 0.0.0.0
 server_options = [{recbuf, 262144}]
 socket_options = [{sndbuf, 262144}, {nodelay, true}]
 require_valid_user = true
+authentication_handlers = {chttpd_auth, cookie_authentication_handler}, {chttpd_auth, proxy_authentication_handler}, {chttpd_auth, default_authentication_handler}
+
+[chttpd_auth]
+require_valid_user = true
+secret = 5ec56423-0856-4b13-8ab7-9b476fea2c39 ; CHANGE ME  - proxy secret
```

Comment:
I also do not think we need to hard-code the `secret` here.  Unless I am mistaken, the [COUCHDB_SECRET envar](https://docs.communityhealthtoolkit.org/hosting/4.x/production/docker/#prepare-environment-variables-file) you can supply to the couch container [gets set as the `httpd_auth.secret` value](https://github.com/medic/cht-core/blob/987947839f1a01538c7abecbfb9ca5b410a60e41/couchdb/docker-entrypoint.sh#L39). 

```suggestion
```